reactioncommerce/reaction
# refactor: update styling of orderCard header section, move details to sidebar

Signed-off-by: Erik Kieckhafer <ek@ato.la>

Part of #5157  
Impact: **minor**  
Type: **refactor**

Style updates for #5157  - update the styling of the header section, and implement the new Sidebar and move some data into the sidebar. 

An incremental update so #5157 isn't 100 files and 2000+ changes.

## Breaking changes
None


## Testing
1. Create an order
1. Go to order page
1. Update URL from `/order/` to `/order-2.0/` to see the new read-only version
1. See that the header area, prior to the fulfillment groups, is updated, and that the sidebar opens with `Order Summary` and `Customer information` when `Show order summary` is clicked

OLD:
![Order_Details_for_order_reference__44000001537](https://user-images.githubusercontent.com/4482263/60228234-e80fe800-9846-11e9-8bc9-9a697d0d02f6.png)

NEW:
![Order_Details_for_order_reference__Rxofk7DZT2kKt4rq6](https://user-images.githubusercontent.com/4482263/60228237-ec3c0580-9846-11e9-831e-cafa8f6db6f5.png)
